frontend/source/translation/fr.po

```diff
@@ -974,3 +974,7 @@ msgstr ""
 #: agrammon/Changelog.js
 msgid "Changelog"
 msgstr ""
+
+#. NO LONGER USED
+msgid "Standard"
+msgstr ""
```

Comment:
If it's no longer used, it can't be removed?

Reply:
removed